namer/marathon/src/main/scala/io/buoyant/namer/marathon/MarathonInitializer.scala

```diff
@@ -64,7 +66,17 @@ case class MarathonConfig(
     case None => 80
   }
   private[this] def getUriPrefix = uriPrefix.getOrElse("")
-  private[this] def getTtl = ttlMs.getOrElse(5000).millis
+
+  @JsonIgnore
+  private[this] def getJitteredTtl(ttl: Int, jitterConf: Int): Stream[Duration] = {
+    require(ttl > jitterConf, "TTL should be greater than jitter")
+    val jitter = (ttl + (scala.util.Random.nextDouble() * 2 - 1) * jitterConf).toInt
```

Comment:
since this is a `val` a value will be computed once and then used for every position in the stream.  I think this should be a `def` so that each position is recomputed.